proto/file/file.proto

```diff
@@ -1,5 +1,7 @@
 syntax = "proto3";
 
+option go_package = "generated";
+
```

Comment:
Why did you put it here?

Reply:
to make the generate proto files work, it generates into a folder.

Reply:
done